pkgs/ok_http/example/android/app/build.gradle

```diff
@@ -29,8 +29,12 @@ android {
     ndkVersion = flutter.ndkVersion
 
     compileOptions {
-        sourceCompatibility = JavaVersion.VERSION_21
-        targetCompatibility = JavaVersion.VERSION_21
+        sourceCompatibility = JavaVersion.VERSION_1_8
```

Comment:
Shouldn't this be also `VERSION_11`?

Reply:
I copied this from the flutter java plugin template without thinking about the values too much. Let me try `VERSION_11`.

Reply:
Works, fixed!